README.md

```diff
@@ -1,7 +1,132 @@
-# WSO2 API Manager 2.5.0 Puppet Modules
+# WSO2 API Manager 2.5.0 Puppet 5 Modules
 
-This repository contains puppet modules for each profile relates to API Manager
+This repository contains puppet modules for each profile relates to API Manager.
 
-## Default profile
-- export FACTER_profile=apim
-- puppet agent -vt
+## Quick Start Guide
+1. Download and copy the `wso2am-linux-installer-x64-2.5.0.deb` or/and `wso2am-linux-installer-x64-2.5.0.rpm` to the files directories in `/etc/puppet/code/environments/dev/modules/__profile__/files` in the Puppetmaster. <br>
+Profile refers to each profile in API Manager <br>
+eg: `/etc/puppet/code/environments/dev/modules/apim/files` <br>
+Dev refers to the sample environment that you can try this modules.
+
+2. Run necessary profile on puppet agent. More details on this is available in following section.
```

Comment:
This line should be >> More details on this "are" available in "the" following section.